internal/util/sessionutil/session_util.go

```diff
@@ -669,7 +672,9 @@ func (s *Session) LivenessCheck(ctx context.Context, callback func()) {
 		panic(err)
 	}
 	s.watchSessionKeyCh = s.etcdCli.Watch(context.Background(), s.getSessionKey(), clientv3.WithRev(getResp.Header.Revision))
```

Comment:
It is recommended to move s.watch SessionKeyChan to a separate function and call it in Register